### ISSUE TITLE
Error instantiating cw721-fixed-price

### ISSUE THREAD

**Issue by interphased** [public-awesome/cw-nfts#43]
Let me preface this issue by saying I'm relatively new to Rust and cosmwasm so this could be "user error."

I was able to mint an NFT on my local development with `cw721-metadata-onchain` but I'm having issues with `cw721-fixed-price`. Yes, all the tests pass, and I'm able to store it, but when I try to instantiate it I get:

```
Error: rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: dispatch: submessages: Error parsing into type cw721_fixed_price::msg::InstantiateMsg: missing field `owner`: instantiate wasm contract failed: invalid request
```

I have tried including `owner` in the json object (and I get errors correctly when not adding `unit_price`, so I know the json is parsing correctly) but that didn't work. I have a feeling it's something with the submessages, maybe incorrect type handling between `Cw721InstantiateMsg` and `InstantiateMsg`. But really I have no idea. Any help would be appreciated. Thanks in advance.

Also, in case versioning matters, I set up my local environment for Juno via the instructions on their site here:
https://docs.junonetwork.io/smart-contracts-and-junod-development/junod-local-dev-setup#run-juno

**Comment by interphased** [comment]
Alright, I understand a bit better now. Submessages are meant for interacting with other contracts, right? My issue was that I was putting the same contact id as the one I was instantiating instead of to a `cw721-base` contract. I am unable to store a `cw721-base` contract though as it doesn't have a required export for instantiate. But when I pointed it to a `cw721-metadata-onchain` contract id it looks like it successfully initialized. So that's a start.

**Comment by Xstoudi** [comment]
Hey! Did you find a solution?

**Comment by JakeHartnell** [comment]
Should be fixed after #49 and #46 are merged.

**Comment by orkunkl** [comment]
Resolved

**Comment by JakeHartnell** [comment]
> Should be fixed after #49 and #46 are merged.

Have been merged, will be fixed in the next tagged release.
